package.json

```diff
@@ -1366,38 +1366,6 @@
                     "default": "",
                     "description": "Equivalent to setting the DOCKER_MACHINE_NAME environment variable."
                 },
-                "docker.importCertificates": {
```

Comment:
Is this the "proper" way to remove / deprecate a setting @EricJizbaMSFT ?

Reply:
Fine with me as long as it's mentioned in the changelog. Also as long as we verify there aren't any major differences between this support and VS Code support (I feel like minor differences could just be filed as issues on VS Code).

Reply:
That's not deprecating, it's removing.  vscode supports a "deprecationMessage" for a setting, you might consider using that for a release before removing.  Also I'm pretty sure I put the usage of this setting into telemetry, so you could see how many people are using it.

Reply:
@StephenWeatherford I think deprecating for a release is reasonable. Part of the motivation for removing is to get rid of a component governance alert. Theoretically it's safe to remove as VSCode's setting is enabled-by-default, but it's also safe to merely deprecate for a release and sit on this PR until then. I'll discuss with PMs too.